tests/quality/test_shrink_quality.py

```diff
@@ -202,20 +202,16 @@ def test_minimize_long():
 
 def test_find_large_union_list():
     def large_mostly_non_overlapping(xs):
-        union = reduce(operator.or_, xs)
+        union = reduce(set.union, xs)
         return len(union) >= 30
 
     result = minimal(
         lists(sets(integers(), min_size=1), min_size=1),
         large_mostly_non_overlapping, timeout_after=120)
     assert len(result) == 1
-    union = reduce(operator.or_, result)
+    union = reduce(set.union, result)
     assert len(union) == 30
     assert max(union) == min(union) + len(union) - 1
-    for x in result:
-        for y in result:
-            if x is not y:
-                assert not (x & y)
```

Comment:
Did you mean to remove this entire assertion?

Reply:
Yep - I've rebased on #1006, which asserts that the result has a single set. There are therefore no combinations of two elements!